test/e2e/scheduling/ubernetes_lite.go

```diff
@@ -148,7 +148,7 @@ func getZoneNameForNode(node v1.Node) (string, error) {
 
 // Return the number of zones in which we have nodes in this cluster.
 func getZoneCount(c clientset.Interface) (int, error) {
-	zoneNames, err := e2enode.GetClusterZones(c)
+	zoneNames, err := e2enode.GetSchedulableClusterZones(c)
```

Comment:
From the comment, it seems like we should include all nodes.

Reply:
Since we reach the agreement, I think all zones should be schedulable ones. Updated.